plugins/nf-google/src/main/nextflow/cloud/google/batch/client/BatchConfig.groovy

```diff
@@ -87,6 +89,7 @@ class BatchConfig {
         result.serviceAccountEmail = session.config.navigate('google.batch.serviceAccountEmail')
         result.retryConfig = new BatchRetryConfig( session.config.navigate('google.batch.retryPolicy') as Map ?: Map.of() )
         result.autoRetryExitCodes = session.config.navigate('google.batch.autoRetryExitCodes', DEFAULT_RETRY_LIST) as List<Integer>
+        result.fuseOptions = session.config.navigate('google.batch.fuseOptions', List.of()) as List<String>
```

Comment:
@JohnWalshTempus can you document this option here:

https://github.com/nextflow-io/nextflow/blob/master/docs/reference/config.md#google

Reply:
I'd propose something like:

`google.batch.fuseOptions`
: :::{versionadded} 25.03.0-edge
  :::
: Defines one or more custom Fuse mount options. Multiple options can be specified using a list object.

Reply:
I've added the docs